kalite/distributed/management/commands/initdconfig.py

```diff
@@ -5,10 +5,12 @@
 
 from django.conf import settings
 from django.core.management.base import BaseCommand
+import getpass
 
-script_template = """#! /bin/sh
+script_template = """
+#!/bin/sh
```

Comment:
Done! This would only be relevant in case output is piped directly into the file... but I guess docs might point out that method.